src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/treasuryopi/TreasuryOpiIngestionWFClient.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.pu.workflow.wf.ingestionflow.treasuryopi;
```

Comment:
let's add a nested level in order to have all treasury WFs under the same folder
```suggestion
package it.gov.pagopa.pu.workflow.wf.ingestionflow.treasury.opi;
```